ref.yml

```diff
@@ -392,29 +399,196 @@ fields:
-              'master' may be replaced by any valid reference like one
-              of the branches available at
-              https://github.com/coq/coq-on-cachix/branches or a valid
-              commit hash.
+            description: Name of the cachix repository
+            examples:
+              - "coq"
+              - "math-comp"
+        - key:
+            required: false
+            description: |
+              Name of GitHub secret variable containing the Cachix signing key.
+              The secret should be generated from https://cachix.org
+              and added to Github, either:
+              - at the repository level
+                https://github.com/OWNER/REPO/settings/secrets/actions
+              - or at the organization level:
+                https://github.com/organizations/ORG/settings/secrets/actions
+              /!\ This is the legacy authentification method and
+              /!\ is discouraged in favor of `token` below.
+            example:
+              - "CACHIX_SIGNING_KEY"
+        - token:
+            required: false
+            description: |
+              Name of GitHub secret variable containing the Cachix auth token.
+              The secret should be generated from https://cachix.org
+              and added to Github, either:
+              - at the repository level
+                https://github.com/OWNER/REPO/settings/secrets/actions
+              - or at the organization level:
+                https://github.com/organizations/ORG/settings/secrets/actions
+            example:
+              - "CACHIX_AUTH_TOKEN"
+  - nix-default:
       used:
-        - .travis.yml
-        - nix-action.yml
+        - default.nix
+        - .nix/coq-nix-toolbox.nix
+      description: >
+        Configure the default.nix to fetch from the generic coq nix toolbox.
+        This will automatically pin the toolbox to the last version of the
+        provided ref (and if no ref is provided, master).
+      fields:
+        - src:
+            required: false
+            type: string
+            description: Sets a local path instead of a github source
+            examples:
+              - "~/git/coq-nix-toolbox/master"
+        - url:
+            required: false
+            type: string
+            description: Change the git url to fetch from
+            default: "https://github.com/coq-community/coq-nix-toolbox.git"
+        - owner:
+            required: false
+            type: string
+            description: Switch to the github fork of another "owner"
+            default: "coq-community"
+        - ref:
+            required: false
+            type: string
+            description: Change the branch of coq-nix-toolbox to clone
+            default: master
+        - rev:
+            required: false
+            type: string
+            description: Change the pinned rev of coq-nix-toolbox
+  - nix-dependencies:
```

Comment:
This used to be provided as the `nix` field of the `dependencies` list. Would it be possible to reuse this (less conversion of projects using the former infrastructure)? If not, we should remove this unused field from the documentation.